packages/block-library/src/button/deprecated.js

```diff
@@ -119,6 +119,150 @@ const blockAttributes = {
 	},
 };
 
+const v11 = {
```

Comment:
Is this deprecation necessary only because we use now `ElementButton.Link` instead of `a`?

Reply:
no because of the extra classname

Reply:
Right, the question is rather if we can relax this validation so it doesn't force us to add another deprecation only because one additional class is necessary in the saved HTML.

Reply:
How?

Reply:
@scruffian  Do we still need this now that all we do is add a class name?

Reply:
that's all we were ever doing